library/Haste/Ajax/ReloadHelper.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+namespace Haste\Ajax;
+
+use Haste\Http\Response\JsonResponse;
+
+class ReloadHelper
+{
+    const TYPE_CONTENT_ELEMENT = 'ce';
+    const TYPE_FRONTEND_MODULE = 'fmd';
```

Comment:
Well those types are used in different classes to identify the entries and that's why they have been introduced. In my opinion it's not convenient that I have to know what string value I need to pass to the method to register a frontend module. Thus the types have been created.